redis/endpoint.bal

```diff
@@ -16,26 +16,32 @@
 
 import ballerina/jballerina.java;
 
-# Represents Redis client endpoint.
+# Ballerina Redis connector provides the capability to access Redis cache.
```

Comment:
Suggestion:
The Ballerina Redis connector provides the capability to access Redis cache.